rotkehlchen/tests/api/test_evm_transactions.py

```diff
@@ -89,3 +89,48 @@ def test_query_transactions(rotkehlchen_api_server: 'APIServer'):
         for entry in result['entries']:
             assert entry['entry']['timestamp'] <= last_ts
             last_ts = entry['entry']['timestamp']
+
+
+@pytest.mark.parametrize('optimism_accounts', [[
+    '0xb8553D9ee35dd23BB96fbd679E651B929821969B',
+]])
+@pytest.mark.parametrize('should_mock_price_queries', [True])
+@pytest.mark.parametrize('default_mock_price_value', [FVal(1.5)])
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+def test_query_transactions_single_chain(rotkehlchen_api_server: 'APIServer'):
+    """
+    Test that querying the evm transactions endpoint for transactions in a single evm chain
+    works even if the address has transactions in other chains too.
```

Comment:
```suggestion
    Test that querying the evm transactions endpoint for an address in a single evm chain
    works even if the address has transactions in other chains too.
```

Reply:
Also for how I understood the error in this test you don't mind that the account has txs in other chain. The important thing here is that you don't have it added as an ethereum account, only optimism. Can you add it here providing more information about what is being tested?

Reply:
About consolidating this test with the `test_query_transactions` I'm not sure, what you would need to do is to remove the ethereum address and also delete at least one op transaction but you also need to update the queried ranges and I believe is too much hustle for testing and can end up wrong. I believe having it as two tests is fine

Reply:
Adjusted the docstring